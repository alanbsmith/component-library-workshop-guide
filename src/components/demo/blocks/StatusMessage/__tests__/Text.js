import { generateSnapshots } from "__tests__/helpers";

import Text from '../Text';

generateSnapshots(Text);
