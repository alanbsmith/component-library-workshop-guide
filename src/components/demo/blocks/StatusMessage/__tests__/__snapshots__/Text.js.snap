// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply bold modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-weight: 700;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply extraSmall modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-size: 0.75rem;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply large modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-size: 1.125rem;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply light modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-weight: 300;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply medium modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-size: 1rem;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply normal modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-weight: 400;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply semiBold modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-weight: 600;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply small modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  font-size: 0.875rem;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply statusColorDanger modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: #DD3800;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply statusColorDefault modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: #9aa0ac;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply statusColorInfo modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: #0975E0;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply statusColorSuccess modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: #008900;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply statusColorWarning modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: #967100;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply text modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: #323f58;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;

exports[`Text should apply textLight modifier 1`] = `
.c0 {
  display: inline-block;
  font-size: 1rem;
  line-height: 1.2em;
  color: #636d7f;
  color: inherit;
  font-size: 0.875rem;
  padding: 0.25rem;
}

<span
  className="c0"
/>
`;
